packages/react-dom-bindings/src/client/DOMNamespaces.js

```diff
@@ -10,31 +10,3 @@
 export const HTML_NAMESPACE = 'http://www.w3.org/1999/xhtml';
```

Comment:
My intuition is that this shouldn't be used anymore because our logic is consistently that we treat HTML as the "catch all" and the only namespaces that are allowed are SVG and MATH.

I think the only place remaining is in `normalizeHTML()` but that should probably check if the namespace is SVG or MATH, otherwise create it without namespace.

That's probably already a bug because it would treat a custom namespace as XML in the comparison which would lead to a mismatch error since everywhere else it's treated as HTML.